.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ secrets.GO_VERSION }}
-    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go get -u github.com/kisielk/errcheck; /home/runner/go/bin/errcheck -tags draft ./...
+    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go build
```

Comment:
Hi @gunjanmimo , I think this line needs to be unedited, we certainly need to check all the errors that can occur at runtime beforehand... This command disables those error checks...